python/paddle/fluid/tests/unittests/dygraph_to_static/test_convert_call.py

```diff
@@ -175,14 +179,106 @@ def test_transformed_static_result(self):
 
 
 class TestThirdPartyLibrary(TestRecursiveCall2):
-    def _run(self):
-        with fluid.dygraph.guard():
-            self.dygraph_func = dyfunc_with_third_library_logging
-            fluid.default_startup_program.random_seed = SEED
-            fluid.default_main_program.random_seed = SEED
-            data = fluid.dygraph.to_variable(self.input)
-            res = self.dygraph_func(data)
-            return res.numpy()
+    def set_func(self):
+        self.dygraph_func = dyfunc_with_third_library_logging
+
+
+# Situation 2 : test not_to_static
+
+
+def func_sum(x):
+    res = paddle.sum(x)
+    return res
+
+
+@paddle.jit.not_to_static
+def func_not_to_static(x):
+    res = func_sum(x)
+    return res
+
+
+@paddle.jit.to_static
+def func_convert_then_not_to_static(x):
+    y = func_not_to_static(x)
+    return y
+
+
+class TestClass(paddle.nn.Layer):
+    @paddle.jit.not_to_static
+    def called_member(self, x):
+        return paddle.sum(x)
+
+    @paddle.jit.to_static
+    def forward(self, x):
+        y = self.called_member(x)
+        return y
+
+
+class TestNotToConvert(TestRecursiveCall2):
+    def set_func(self):
+        self.dygraph_func = func_not_to_static
+
+    def test_CONVERSION_OPTIONS(self):
```

Comment:
Naming should be `test_conversion_options`

Reply:
Thanks, done.